assets/js/controllers/product/product-with-variations.js

```diff
@@ -12,6 +12,10 @@ storefrontApp.controller('productWithVariationsController', ['$scope', '$window'
         $scope.filterableVariationPropsMap = { };
         $scope.selectedVariation = {};
 
+        $scope.variationsQuantities = undefined;
+        $scope.totalPrice = "$0";
```

Comment:
Can we use the same approach we did here? That way we'll be able to work with different currencies
https://github.com/VirtoCommerce/vc-demo-theme-b2b/blob/0b29779ce376a450008d962030715dbba3cf0f2c/assets/js/checkout/checkout-line-item.js#L42